template-tanstack-start-clerk/convex/auth.config.ts

```diff
@@ -6,7 +6,7 @@ export default {
       // and configure CLERK_JWT_ISSUER_DOMAIN on the Convex Dashboard
       // See https://docs.convex.dev/auth/clerk#configuring-dev-and-prod-instances
       // Should look similar to 'https://main-swine-30.clerk.accounts.dev'.
-      domain: process.env.CLERK_JWT_ISSUER_DOMAIN,
+      domain: process.env.VITE_CLERK_FRONTEND_API_URL,
```

Comment:
This doesn’t look correct since this is evaluated on the Convex backend. Did you mean to do this change?